CMakeLists.txt

```
@@ -11,6 +11,7 @@ set(bpftrace_VERSION_PATCH 0)
 include(GNUInstallDirs)
 
 set(WARNINGS_AS_ERRORS OFF CACHE BOOL "Build with -Werror")
+set(HARDENED_STDLIB OFF CACHE BOOL "Enable hardened definitions for standard library")
```

seems like these options are for development use only? not sure `_LIBCPP_HARDENING_MODE_DEBUG` is suitable for production usage according to the docs.
```suggestion
set(HARDENED_STDLIB OFF CACHE BOOL "Enable hardened definitions for standard library (for development use only)")
```